charts/brigade-project/values.yaml

```diff
@@ -68,3 +68,12 @@ allowPrivilegedJobs: "true"
 # OPTIONAL: Use this to allow host mounted docker sockets in your jobs.
 # This is a big security risk if your project is public-facing; enable at your own risk.
 # allowHostMounts: "true"
+
+# Kubernetes-specific configuration options.
+kubernetes: {}
```

Comment:
Nit but this seems to work without `{}`. Then, perhaps we'd rather like to remove it, so that you can just uncomment any of the settings below to start customizing values?